app/src/main/java/com/gh4a/fragment/UserFragment.java

```diff
@@ -52,11 +52,16 @@
 import java.util.Map;
 
 public class UserFragment extends LoadingFragmentBase implements View.OnClickListener {
+    private static final String LOGIN = "login";
+    private static final String SORT = "sort";
+    private static final String PUSHED = "pushed";
+    private static final String AFFILIATINO = "affiliation";
```

Comment:
Typo.

Also, the filter strings should live in the Github API bindings.

Reply:
you meant "AFFILIATINO" spelling error right ?

Reply:
> the filter strings should live in the Github API bindings. 

I created a **Constants** file

Reply:
> I created a Constants file

Plesse change that to either use the existing IGithubConstants class or put the constants to the respective service classes. If the constants only apply to one service, I'd prefer the latter.